arbos/arbosState/initialize.go

```diff
@@ -190,7 +195,7 @@ func initializeRetryables(statedb *state.StateDB, rs *retryables.RetryableState,
 			return err
 		}
 		if r.Timeout <= currentTimestamp {
-			statedb.AddBalance(r.Beneficiary, uint256.MustFromBig(r.Callvalue))
+			statedb.AddBalance(r.Beneficiary, uint256.MustFromBig(r.Callvalue), tracing.BalanceChangeUnspecified)
```

Comment:
do we want to create new BalanceChangeReasons?
Can be in a different PR.

Reply:
created a separate issue for that

Reply:
Resolving the comment in favour of new PR